_data/investing.yml

```diff
@@ -33,6 +33,12 @@ websites:
       exceptions:
           text: "2FA only available on investment accounts. Not available for 401K plans."
 
+    - name: HealthEquity
+      url: https://www.healthequity.com/
```

Comment:
Is there a reason for me not being able to connect to the site? Neither with https nor http